### DISCUSSION TITLE
Adding in computestacks installer.

### DISCUSSION THREAD

**Discussion by CloudRack-Development** [tteck/Proxmox#3981] [Ideas]
adding the ability of Adding in computestacks installer.

https://github.com/ComputeStacks/terraform-cs-proxmox

computestacks found -> https://computestacks.com

**Comment by tteck** [comment]
I would _think_ that project is dead. Over a year without a commit!

**Reply by CloudRack-Development** [reply]
_i think_ as the worlds in hard times things die off and come back alot of the time as well tho
